admin/worker_4tests.m.template

```diff
@@ -13,14 +13,39 @@ function [ok,err_mess,je]=worker_4tests(worker_controls_string)
 %              contains only minimal initialization information, namely the
 %              folder name where the job initialization data are located on
 %              a remote system.
-
+je = [];
 if isempty(which('herbert_init.m'))
     herbert_on();
-    % Set up non-default herbert or Horace path to test them
-    % herbert_on('/home/isis_direct_soft/Herbert_bugfix/herbert_core');
-    % horace_on('/home/isis_direct_soft/Horace_bugfix/horace_core');
+    %horace_on('c:\Users\abuts\Documents\developing_soft\Horace_3_4\horace_core');
```

Comment:
Comment left here.

Also, this `herbert_on` call should be within the `try-catch` below. If `herbert_on` fails we could end up leaving processes lying around

Reply:
done